xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/transactions/UnlModify.java

```diff
@@ -62,7 +62,7 @@ static ImmutableUnlModify.Builder builder() {
    */
   @Override
   @JsonProperty("Account")
-  @Value.Derived
+  @Value.Default
```

Comment:
related: https://github.com/XRPLF/xrpl4j/issues/590

Reply:
Nice catch - I'll take a look at this one tonight.

Reply:
Note this should be fixed in [v4.0.5](https://github.com/XRPLF/xrpl4j/releases/tag/v4.0.5)